xdsl/dialects/experimental/dmp.py

```diff
@@ -302,7 +311,7 @@ def from_index_attrs(
     ):
         data_type = builtin.i64
         return ShapeAttr(
-            [
+            *[
```

Comment:
I think `*( ... )` will avoid creating the list.

Reply:
probably will I'll make the change

Reply:
Done